app/constants/view.ts

```diff
@@ -28,6 +28,8 @@ export const CURRENT_CALL_BAR_HEIGHT = 74;
 
 export const QUICK_OPTIONS_HEIGHT = 270;
 
+export const MENU_ITEM_HEIGHT = 50;
```

Comment:
If we are removing `MenuItem`, does it make sense having something like `MENU_ITEM_HEIGHT`? Shouldn't it be something like `OPTION_ITEM_HEIGHT` in any case?